### ISSUE TITLE
deposited btc no etherum 24hrs now

### ISSUE THREAD

**Issue by grtt** [ethereum/mist#2063]
#### System information
``````
Version: `0.0.0`
OS & Version: windows/linux/osx
Node type: `eth/geth(default)`
``````

#### Please check the **already existing** issues to keep duplicates at a minimum.
Furthermore several **work-arounds** have been collected in the [Mist-Troubleshooting-Guide](https://github.com/ethereum/wiki/wiki/Mist-Troubleshooting-Guide).

If possible add the following to your report:

- Screenshots

- Check the console, of Mist (`CTRL/CMD + ALT + i`) and take a screenshot

- Log files 
  - go to `menu -> accounts -> backup -> application data`
  - zip and upload `node.log` and all other `node.log.X` files

**Comment by evertonfraga** [comment]
Hello @grtt. Is your local blockchain synced? 
In order to confirm your balance, before it has finished to sync, you can hop into a blockchain explorer like this, and search for your Ethereum address.

If, by any chance, your balance is zeroed, you should get in touch with the service you bought the ether from (e.g. coinbase, shapeshift, ...).

Example: https://etherscan.io/address/0x33be28a46b6dc3211350a747d821eb6a2ba089de

![image](https://cloud.githubusercontent.com/assets/47108/24358212/cb7d5e04-12d6-11e7-86f3-822f1bcb2d8f.png)

**Comment by grtt** [comment]
Hi transfer confirmed say I have balance in my account not true

**Comment by BitcoinsNinja** [comment]
I think a lot of us are doing one mistake here: Not creating an ETH wallet to store our ETH.

If you check your main ETH balance by clicking "Main Account (EtherBase), it gives you the following message:

NOTE

Accounts can't display incoming transactions, but hold and send ether. To see incoming transactions create a wallet contract to store ether.

So, before trading your BTC to procure ETH, DO NOT forget to create your personal wallet to store your ETH.

May be thats where we are going wrong.

Hope this helps.

Vikas Solanki
BitcoinsNinja.com

**Comment by luclu** [comment]
Closed as duplicate of #2070.
